docs/index.md

```diff
@@ -483,7 +487,7 @@ Template customization:
                         A file with kwargs for custom formatters.
   --custom-template-dir CUSTOM_TEMPLATE_DIR
                         Custom template directory
-  --encoding ENCODING   The encoding of input and output (default: utf-8)
```

Comment:
Why has this line has changed? I can't find any edit related to default file encoding

Reply:
Yeah the CI is breaking because of this, my guess is that the readme generator behaves differently on Windows and UNIX, and the PR creator ran it on Windows causing this change.

Reply:
OT: I wonder if we should set a default encoding instead of relying on the system default